.gitmodules

```diff
@@ -1,21 +1,21 @@
 [submodule "_gallery"]
 	path = _gallery
-	url = https://www.github.com/Second-Time-Is-The-Charm/_gallery
+	url = https://www.github.com/Angeloth1/_gallery
 [submodule "_quotes"]
 	path = _quotes
-	url = https://www.github.com/Second-Time-Is-The-Charm/_quotes
+	url = https://www.github.com/Angeloth1/_quotes
 [submodule "images"]
 	path = images
-	url = https://www.github.com/Second-Time-Is-The-Charm/images
+	url = https://www.github.com/Angeloth1/images
 [submodule "_bibliography"]
 	path = _bibliography
-	url = https://www.github.com/Second-Time-Is-The-Charm/bibliography
+	url = https://www.github.com/Angeloth1/bibliography
 [submodule "_notes"]
 	path = _notes
-	url = https://www.github.com/Second-Time-Is-The-Charm/notes
+	url = https://www.github.com/Angeloth1/notes
 [submodule "_includes/extras"]
 	path = _includes/extras
-	url = https://www.github.com/Second-Time-Is-The-Charm/extras
+	url = https://www.github.com/Angeloth1/extras
 [submodule "_includes/text"]
 	path = _includes/text
-	url = https://www.github.com/Second-Time-Is-The-Charm/text
+	url = https://www.github.com/Angeloth1/text
```

Comment:
Χρειάζεται αλλαγές στο link πρέπει να δείχει το site

Reply:
Fixed: submodules(Second-Time-Is-The-Charm)